src/main.rs

```diff
@@ -128,10 +131,17 @@ pub fn list_languages(assets: &HighlightingAssets, term_width: usize) {
     }
 }
 
-pub fn list_themes(assets: &HighlightingAssets) {
+pub fn list_themes(assets: &HighlightingAssets, config: &mut Config) {
```

Comment:
I was thinking that `list_themes` could take a non-mutable reference to `Config` and could then  create a mutable copy by calling `.clone()` in the body of the function.